spotrec.py

```diff
@@ -49,6 +51,7 @@
 _shell_executable = "/bin/bash"  # Default: "/bin/sh"
 _shell_encoding = "utf-8"
 _ffmpeg_executable = "ffmpeg"  # Example: "/usr/bin/ffmpeg"
+_pl_track_counter = 1
```

Comment:
This variable is placed under the "# Hard-coded settings" section. This is meant for constants. It should be moved down to "# Variables that change during runtime"